neo/app/assets/javascripts/lib/dc.leaflet.js

```diff
@@ -0,0 +1,434 @@
+dc.leafletChart = function(_chart) {
```

Comment:
Could you rename all those variable names without the leading `_`? We don't have this convention before. It's better to not introducing a new naming rule.